hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieBaseRelation.scala

```diff
@@ -420,7 +421,9 @@ abstract class HoodieBaseRelation(val sqlContext: SQLContext,
           inMemoryFileIndex.listFiles(partitionFilters, dataFilters)
         }
 
-        val fsView = new HoodieTableFileSystemView(metaClient, timeline, partitionDirs.flatMap(_.files).toArray)
+        // fail time travel query when covers hollow commit
+        val completedTimeline = handleHollowCommitIfNeeded(timeline, metaClient, HollowCommitHandling.FAIL)
```

Comment:
can we add tests for diff HollowCommitHandling ?

Reply:
covered in org.apache.hudi.functional.TestTimeTravelQuery#testTimeTravelQueryWithIncompleteCommit